wonnx/templates/pool/conv.wgsl

```diff
@@ -28,18 +28,21 @@ fn main(@builtin(global_invocation_id) global_id: vec3<u32>) {
 
 		let m = rest / {{ o_chunks[0][1] }}u;
 		rest = rest % {{ o_chunks[0][1] }}u;
-		
+
 		let y = rest / {{ o_chunks[0][2] }}u;
 		let x = rest % {{ o_chunks[0][2] }}u;
-		
+
+		let M = {{ o_shape[0][1] }}u;
```

Comment:
Not very Rusty to use single-letter upper case variable names, but I assume this aligns with the ONNX spec? If not consider making this more descriptive

Reply:
Yeah exactly. It was an attempt at helping me understand how this all works exactly.